Sources/Container.swift

```diff
@@ -224,6 +238,10 @@ extension Container: _Resolver {
             var resolvedInstance: Service?
             let key = ServiceKey(serviceType: Service.self, argumentsType: Arguments.self, name: name, option: option)
 
+            if key == Self.graphIdentifierKey {
+                return currentObjectGraph as! Service?
```

Comment:
```suggestion
                return currentObjectGraph as? Service
```
Has similar runtime performance